dev-docker-compose.yml

```
@@ -172,11 +177,124 @@ services:
       - app-network
     restart: always
 
+
+  traefik:
+    image: traefik:v2.9
+    command:
+      - "--api.insecure=true"
+      - "--providers.docker=true" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)